rllib/policy/policy.py

```diff
@@ -1536,3 +1565,25 @@ def __repr__(self):
     @Deprecated(new="get_exploration_state", error=True)
     def get_exploration_info(self) -> Dict[str, TensorType]:
         return self.get_exploration_state()
+
+
+def get_gym_space_from_struct_of_tensors(value: TensorStructType) -> gym.spaces.Dict:
+    value_dict = NestedDict(value)
```

Comment:
Shouldn't we generalize this to also accept a single Tensor (not inside a dict/tuple)?

Reply:
done.